### ISSUE TITLE
About rgb_watson_vgg_trial0.pth

### ISSUE THREAD

**Issue by qileee** [facebookresearch/stable_signature#45]
Hi， Thanks for your job.  When I tried to reproduce this work, I found that the finetune_ldm_decoder.py code used `watson_vgg` as the loss function and `pretrain=True`, but I couldn't find the corresponding file in `src/loss/losses/` folder. Could you help me where can i get it. Thank you!
